packages/excalidraw/element/binding.ts

```diff
@@ -756,7 +759,10 @@ export const fixBindingsAfterDuplication = (
   sceneElements
     .filter(({ id }) => allBindableElementIds.has(id))
     .forEach((bindableElement) => {
-      const { boundElements } = bindableElement;
+      const oldElementId = duplicateIdToOldId.get(bindableElement.id);
+      const { boundElements } = sceneElements.filter(
+        ({ id }) => id === oldElementId,
+      )[0];
```

Comment:
let's use `find` so we don't alloc an array for no reason (will require type assertion `!`, but once we check for unsafe array access we'd need it anyway)